custom_mutators/libfuzzer/FuzzerLoop.cpp

```diff
@@ -206,6 +206,8 @@ void Fuzzer::StaticDeathCallback() {
 
 void Fuzzer::DumpCurrentUnit(const char *Prefix) {
 
+  return;
```

Comment:
What's the reason for an early return here? Just to make syncing with upstream libfuzzer easier?

Reply:
makes the diff for updates easier